splice_machine/src/main/java/com/splicemachine/derby/utils/SpliceAdmin.java

```diff
@@ -1476,7 +1479,11 @@ public static void SET_MIN_RETENTION_PERIOD(String schemaName, String tableName,
             DependencyManager dm = dd.getDependencyManager();
             dm.invalidateFor(td, DependencyManager.ALTER_TABLE, lcc);
             tc.prepareDataDictionaryChange(DDLUtils.notifyMetadataChange(ddlChange));
-            td.setMinRetainedVersions(retentionPeriod);
+            if(td.purgeDeletedRows()) {
```

Comment:
This warning could/should be display in sqlshell.
On top of that, we could add an extra warning if the hbase class is not set to SpliceDefaultFlusher

Reply:
> we could add an extra warning if the hbase class is not set to SpliceDefaultFlusher

created a [JIRA ticket](https://splicemachine.atlassian.net/browse/DB-10263) for that.